src/io/flutter/run/FlutterPositionMapper.java

```diff
@@ -196,9 +197,6 @@ private String findRemoteSourceRoot(String remotePath) {
       results.add(threeSlashize(new File(file.getPath()).toURI().toString()));
     }
 
-    // straight path - used by some VM embedders
-    results.add(file.getPath());
```

Comment:
@devoncarew Should I remove similar line from the Dart plugin as well?
It was added by this [commit](https://github.com/JetBrains/intellij-plugins/pull/423/commits/60df87d36f2d85aed2bab720e4ca3ac34fb6d93b) but I'm not sure how useful it was/is.

Reply:
Thanks, I'll remove.
If there are any other improvements/fixes that are not mirrored in the Dart plugin yet, PRs are welcome!

Reply:
For sure; I do try and move over relevant items or file issues (don't always capture all the Flutter plugin changes).